specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -419,6 +427,13 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "query",
+            "name": "includeAll",
```

Comment:
>includeAll [](http://example.com/codeflow?start=21&length=10)

Can you do this with a $select parameter?

Reply:
Please respond.

Some additional details are here : 

[https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options)